web/explorer/src/components/DescriptionPanel.vue

```diff
@@ -1,15 +1,29 @@
 <template>
-    <div class="flex flex-column align-items-center">
+    <div class="flex flex-column align-items-center mb-6">
         <div class="text-center">
             <h1>
                 <img src="@/assets/images/logo-full.png" alt="Capa: identify program capabilities" />
-                <h6 class="font-medium" style="color: rgb(176, 26, 26)">capa: identify program capabilities</h6>
+                <h3 class="font-medium" style="color: rgb(176, 26, 26)">capa: identify program capabilities</h3>
             </h1>
         </div>
         <div>
-            <p class="text-xl max-w-75rem" style="max-width: 75ch">
-                Capa-WebUI is a web-based tool for exploring the capabilities identified in a program. It can be used to
-                search and display the rule matches in different viewing modes.
+            <p class="text-xl p-3 max-w-75rem" style="max-width: 75ch">
+                capa explorer web is a web-based tool for exploring the capabilities identified in a program. It can be
+                used to search and display the rule matches in different viewing modes.
+            </p>
+        </div>
+        <div class="bg-blue-50 border-round p-3 text-center max-w-75rem">
+            <p class="text-900 text-sm m-0">
+                New to capa? Explore the
+                <a
+                    href="https://github.com/mandiant/capa"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                    class="text-blue-600 hover:text-blue-400"
+                >
+                    capa GitHub repository
+                </a>
+                to learn how to generate analysis results.
```

Comment:
nice!

Reply:
very cool, should we also add here the super quick start guide?
- run `capa sample.exe > results.json`
- upload `results.json`
- enjoy

Reply:
I added the getting start steps: https://s-ff.github.io/capa/explorer/

Reply:
awesome!